libsolidity/codegen/ir/IRGenerationContext.cpp

```diff
@@ -38,6 +38,41 @@ ContractDefinition const& IRGenerationContext::mostDerivedContract() const
 	return *m_mostDerivedContract;
 }
 
+ContractDefinition const* IRGenerationContext::superContract(ContractDefinition const& _contract) const
```

Comment:
This can be part of ContractDefinition with passing mostDerivedContract as parameter